src/task2/repository/usersRepository.js

```diff
@@ -0,0 +1,20 @@
+const dataBase = new Map();
+
+async function saveUser(user) {
+    dataBase.set(user.id, user);
+    return user;
+}
+
+async function getUser(id) {
+    return dataBase.get(id);
+}
+
+async function getAll() {
+    return dataBase.values();
```

Comment:
note that logically you should not return isDeleted users

also note that you should return user/users without password field due to security

Reply:
Fixed